google_analytics/aggregation.py

```diff
@@ -182,16 +318,16 @@ def one_hot_encode_categoricals(data, categorical_columns):
     ------
     The one-hot encoded DataFrame.
     """
-    for col in categorical_columns:
+    OHE = ['operatingSystem', 'country', 'browser', 'weekday']  # noqa
+    for col in OHE:
         if data[col].dtypes in ["int64", "float64"]:
             warnings.warn("Column {} converted to from numeric to category".format(col))
             data[col] = data[col].astype("category")
-    if categorical_columns == ['nr_months_ago']:
-        encoded = pd.get_dummies(data[categorical_columns], prefix=['nr_visits'])
-    else:
-        encoded = pd.get_dummies(data[categorical_columns], prefix=categorical_columns)
-    encoded["fullVisitorId"] = data["fullVisitorId"]
-    return encoded.groupby("fullVisitorId").sum()
+        print('Creating the OHE of {}'.format(col))
+        # create categories
+        data = pd.concat([data, pd.get_dummies(data[col], prefix='category_')], sort=False)
```

Comment:
Here the dataset is concatenated on the wrong axis. Axis should be 1, instead of 0 by default.